.github/ISSUE_TEMPLATE/general-issue.yml

```diff
@@ -0,0 +1,16 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: General Issue
+description: A blank issue for general purpose use.
+title: "[Topic] Title"
```

Comment:
Maybe `[component] Title`? Usually we write the component in lower case rather than title case so this would be a hint at that.